internal/confmapprovider/discovery/bundle/bundle.d/receivers/rabbitmq.discovery.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+{{ receiver "rabbitmq" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)rabbitmq.*"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)rabbitmq.*" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)rabbitmq.*"
+  config:
+    default:
+      endpoint: '`endpoint`'
+      username: {{ defaultValue }}
+      password: {{ defaultValue }}
+      collection_interval: 10s
+  status:
+    metrics:
+      - status: successful
+        strict: rabbitmq.consumer.count
+        message: rabbitmq receiver is working!
+    statements:
+      - status: failed
+        regexp: 'connect: network is unreachable'
+        message: The container cannot be reached by the Collector. Make sure they're in the same network.
+      - status: failed
+        regexp: 'connect: connection refused'
+        message: The container is refusing kafka server connections.
```

Comment:
not kafka... What casing/conventions are we using for these messages? Brand name, e.g.: RabbitMQ, or just lowercase always?

Reply:
Can't this also happen with the `host_observer`?

Reply:
copy pasta :)

Reply:
good catch

Reply:
Brand name but for matching we need to use the exact receiver names ( component.go)